x/margin/client/cli/tx_update_params.go

```diff
@@ -44,31 +43,115 @@ func CmdUpdateParams() *cobra.Command {
 				return err
 			}
 
-			leverage_max := sdk.MustNewDecFromStr(viper.GetString("leverage-max"))
+			leverage_max, err := cmd.Flags().GetString("leverage-max")
+			if err != nil {
+				return err
+			}
+
+			interest_rate_max, err := cmd.Flags().GetString("interest-rate-max")
+			if err != nil {
+				return err
+			}
+
+			interest_rate_min, err := cmd.Flags().GetString("interest-rate-min")
+			if err != nil {
+				return err
+			}
+
+			interest_rate_increase, err := cmd.Flags().GetString("interest-rate-increase")
+			if err != nil {
+				return err
+			}
+
+			interest_rate_decrease, err := cmd.Flags().GetString("interest-rate-decrease")
+			if err != nil {
+				return err
+			}
+
+			health_gain_factor, err := cmd.Flags().GetString("health-gain-factor")
+			if err != nil {
+				return err
+			}
+
+			epoch_length, err := cmd.Flags().GetInt64("epoch-length")
+			if err != nil {
+				return err
+			}
+
+			removal_queue_threshold, err := cmd.Flags().GetString("removal-queue-threshold")
+			if err != nil {
+				return err
+			}
+
+			max_open_positions, err := cmd.Flags().GetInt64("max-open-positions")
```

Comment:
can you remove snake case variable names?

Reply:
@jelysn Done.